.changeset/cold-apes-leave.md

```diff
@@ -0,0 +1,6 @@
+---
+"@twilio-paste/chat-composer": patch
+"@twilio-paste/core": patch
+---
+
+[ChatComposer] updated JS Doc used for tpye gen in docs website
```

Comment:
suggestion(non-blocking): no change required here, just a little tip for future changelogs, I like to be super specific to let our consumers know why we change certain files, like specifying that it was just a typo fix to a prop description